crowbar_framework/app/models/service_object.rb

```diff
@@ -758,7 +774,7 @@ def _proposal_update(bc_name, inst, proposal, validate_after_save = true)
       [200, {}]
     rescue Net::HTTPServerException => e
       Rails.logger.error ([e.message] + e.backtrace).join("\n")
-      [e.response.code, {}]
+      [e.response.code, e.message]
```

Comment:
I would guess so, although I'm not really sure, tbh. My gut feeling is that the chef errors should not be transmitted to the user, but maybe I'm wrong.